### PR TITLE
Fixed indexation of storage traits

### PR DESCRIPTION
**Pull request by xgreenx**
Inherited InkStorage for all contracts.
Created STUB derive macros to enable indexing of traits.

### CODE REVIEW

File: contracts/access/access-control/Cargo.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ scale-info = { version = "0.6", default-features = false, features = ["derive"],
 
 strum_macros = "0.21.1"
 brush = { path = "../../brush", default-features = false }
+access_controll_macro = { path = "./macro" }
```
Comment (VargSupercolony) [inline contracts/access/access-control/Cargo.toml:20]: change access_controll_macro to access_control_macro - a little spelling error

File: contracts/token/psp1155/macro/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[package]
+name = "psp1155_macro"
```
Comment (VargSupercolony) [inline contracts/token/psp1155/macro/Cargo.toml:2]: pls rename to psp1155_derive - that's the standard naming of derive macro crates

File: contracts/access/access-control/macro/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[package]
+name = "access_controll_macro"
```
Comment (VargSupercolony) [inline contracts/access/access-control/macro/Cargo.toml:2]: rename to access_control_derive

File: contracts/token/psp20/macro/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+[package]
+name = "psp20_macro"
```
Comment (VargSupercolony) [inline contracts/token/psp20/macro/Cargo.toml:2]: rename to psp20_derive

File: contracts/token/psp721/macro/Cargo.toml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+[package]
+name = "psp721_macro"
```
Comment (VargSupercolony) [inline contracts/token/psp721/macro/Cargo.toml:2]: rename psp721_macro -> psp721_derive

**Review by VargSupercolony — CHANGES_REQUESTED** [review]
naming suggestions

**Review by VargSupercolony — APPROVED** [review]
great work: created derive macros for traits to fix indexing and make code completion (presentation compiler) work

**Review by PierreOssun — APPROVED** [review]
Ok for me

**Pull request merged**